### PR TITLE
Add metric request issue template

### PR DESCRIPTION
**Pull request by hemajv**
Adding a metric request issue template to better assist various teams on their KPI/metric needs.

Related: #10

### CODE REVIEW

**Review by oindrillac — APPROVED** [review]
/lgtm

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm

**Review by schwesig — COMMENTED** [review]
/lgtm
/approve

File: .github/ISSUE_TEMPLATE/metric_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Metric Request
+about: Create a request for metrics/KPIs to be measured
+labels: metric
+---
+
+**Is your metric request related to a problem? Please describe.**
```
Comment (MichaelClifford) [inline .github/ISSUE_TEMPLATE/metric_request.md:7]: Is this section needed? Do we typically think of metrics as being associated with a problem other than "Its a problem I don't have access to this metric?" Maybe add a section "associated OKRs" or include that instruction in the Definition section below?  WDYT?

**Comment by MichaelClifford** [comment]
@hemajv can you fix the pre-commit issues please :)

File: .github/ISSUE_TEMPLATE/metric_request.md (hunk 1 above)
Comment (hemajv) [reply .github/ISSUE_TEMPLATE/metric_request.md:7]: yes makes sense, Ill include it in the definition section :+1:

**Comment by hemajv** [comment]
> @hemajv can you fix the pre-commit issues please :)

I think it should be fixed once https://github.com/open-services-group/metrics/pull/25 is merged

**Review by Shreyanand — APPROVED** [review]
/approve

**Pull request merged**